### ISSUE TITLE
Join the Community

### ISSUE THREAD

**Issue by Susmita-Dey** [WeMakeDevs/github-tutorial#1067]
I want to join this organization here. Kindly create an issue so that other people like me could also join this community by creating an issue and get invited and involved in the community. This will engage more people to get involved into Open Source.
Thanks in advance.

**Comment by abs007** [comment]
True. I believe that there should be sample good first issues here so that users learning can experiment hands on. Doing this would lessen the gap between learning and actually applying your skills.

**Comment by Gokilp** [comment]
I also joined  the community last few month  more useful and  community  work are awesome his will engage more people to get involved into Open Source.
Thanks in advance.
